Tests/UnitTests/SubscriberAttributes/PurchasesSubscriberAttributesTests.swift

```diff
@@ -256,7 +272,7 @@ class PurchasesSubscriberAttributesTests: TestCase {
 
         self.mockNotificationCenter.fireNotifications()
 
-        expect(self.mockSubscriberAttributesManager.invokedSyncAttributesForAllUsersCount) == 2
```

Comment:
This was only 2 because `fireNotifications` was firing the `didEnterForeground` and `didEnterBackground`, and this test was detecting the double update.
Now it's `1` because it only gets updated automatically (regardless of throttling) when entering the background.